Docs/pages/usage/verifying.md

```diff
@@ -0,0 +1,92 @@
+# Verifying Stone Proof Using the Binary
+
+This guide will walk you through how to verify a Stone proof using the provided binaries. It includes steps for both setting up the environment using Docker and manually installing the required dependencies.
+
+## Creating and Verifying a Proof of a Cairo Program
```

Comment:
Please, check if you need cairo-vm for this task

Reply:
@dmirgaleev  Yes, I checked and the Cairo-vm is needed.